app/services/report.py

```diff
@@ -62,3 +61,20 @@ def add_report(self, twitter_id: str, reporter_id: str) -> Report:
 
         report_db.save()
         return report_db
+
+    def process_report(self, twitter_id: str, method: str):
+        report_db = Report.objects(
+            report_key__twitter_id=twitter_id, expired=False
+        ).first()
+        full_details = full_details = self.twitter_scraper.get_full_details(
+            twitter_id, tweets_num=2
+        )
+        label = int()
+        if method == "approve":
+            label = 0 if report_db.score >= 0.5 else 1
+        elif method == "reject":
+            label = 1 if report_db.score >= 0.5 else 0
```

Comment:
If moderator reject the report, I thought we just delete the report? Rejecting a report means denying an opposite label for user, not confirming the current label, right?

Reply:
![image](https://user-images.githubusercontent.com/57407829/165239063-c38be773-8c38-4183-bfd4-d08e127fc42f.png)
Dũng said this so i confirm the current label. But it's our choice to design that.